stable/dask-distributed/README.md

```diff
@@ -0,0 +1,87 @@
+# Dask Distributed Helm Chart
+
+Dask Distributed allows distributed computation in Python the chart also includes a single user Jupyter Notebook
+
+* https://github.com/dask/distributed
+* http://jupyter.org/
+
+## Chart Details
+This chart will do the following:
+
+* 1 x Dask Scheduler with port 8786 (scheduler) and 8786 (webUI) exposed on an external LoadBalancer
```

Comment:
Yes, this is a copy paste error it was supposed to be 8787. I changed it to 80, i think its ok to use 80 for the UI since it uses a `LoadBalancer` service type.